core/src/test/java/greencity/controller/OrderControllerTest.java

```diff
@@ -224,6 +232,22 @@ void updatesRecipientsInfo() throws Exception {
         verify(ubsClientService).updateUbsUserInfoInOrder(ubsCustomersDtoUpdate, null);
     }
 
+    @Test
+    void updatesRecipientsInfoWithOutUser() throws Exception {
+        UbsCustomersDto ubsCustomersDto = getUbsCustomersDto();
+        UbsCustomersDtoUpdate ubsCustomersDtoUpdate = getUbsCustomersDtoUpdate();
+        when(ubsClientService.updateUbsUserInfoInOrder(ubsCustomersDtoUpdate, null)).thenReturn(ubsCustomersDto);
+        lenient().when(ubSuserRepository.findById(ubsCustomersDtoUpdate.getRecipientId())).thenReturn(Optional.empty());
+        ObjectMapper objectMapper = new ObjectMapper();
+        mockMvc.perform(put(ubsLink + "/update-recipients-data")
+            .contentType(MediaType.APPLICATION_JSON)
+            .content(objectMapper.writeValueAsString(ubsCustomersDtoUpdate))
+            .principal(principal))
+            .andExpect(status().isOk());
```

Comment:
this test is not valid
we expect `UBSuserNotFoundException` when no user
see https://github.com/ita-social-projects/GreenCityUBS/blob/dev/service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java#L869